x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/operator/MvExpandOperatorStatusTests.java

```diff
@@ -35,20 +35,22 @@ protected Writeable.Reader<MvExpandOperator.Status> instanceReader() {
 
     @Override
     public MvExpandOperator.Status createTestInstance() {
-        return new MvExpandOperator.Status(randomNonNegativeInt(), randomNonNegativeInt());
+        return new MvExpandOperator.Status(randomNonNegativeInt(), randomNonNegativeInt(), randomNonNegativeInt());
     }
 
     @Override
     protected MvExpandOperator.Status mutateInstance(MvExpandOperator.Status instance) {
         switch (between(0, 1)) {
```

Comment:
This should change to `between(0, 2)` and there should be an arm that just changes pagesIn and another that just changes pagesOut.

Reply:
Makes sense, doing it now